bedrock/firefox/templates/firefox/browsers/index.html

```diff
@@ -0,0 +1,151 @@
+          <!-- TODO <li class="mzp-c-menu-list-item"><button>Email me a link</button></li> -->
+        </ul>
+      </div>
+      <p><a class="mzp-c-cta-link" href="{{ url('firefox.mobile') }}">{{ _('Learn more') }}</a></p>
+    </div>
+    <div class="c-landing-grid-item">
+      {{ high_res_img('img/firefox/browsers/enterprise.jpg', {'alt': '', 'class': 'c-landing-grid-img', 'width': '327', 'height': '223'}) }}
+      <h2 class="c-landing-grid-title"><a href="{{ url('firefox.enterprise.index') }}">{{ _('Enterprise') }}</a></h2>
+      <p>{{ _('Get unmatched data protection with support cycles tailored to suit your company’s needs.') }}</p>
+      <p><a class="mzp-c-cta-link" href="{{ url('firefox.enterprise.index') }}">{{ _('Learn more') }}</a></p>
+    </div>
+    <div class="c-landing-grid-item">
+      {{ high_res_img('img/firefox/browsers/reality.jpg', {'alt': '', 'class': 'c-landing-grid-img', 'width': '327', 'height': '223'}) }}
+      <h2 class="c-landing-grid-title"><a href="https://mixedreality.mozilla.org/firefox-reality{{ referrals }}">{{ _('Reality') }}</a></h2>
+      <p>{{ _('Go beyond two dimensions and enjoy the best immersive content from around the web.') }}</p>
+      <p><a class="mzp-c-cta-link" href="https://mixedreality.mozilla.org/firefox-reality{{ referrals }}">{{ _('Learn more') }}</a></p>
+    </div>
+    <div class="c-landing-grid-item c-landing-grid-wide">
+      <div class="t-dev">
+        {% call feature_card(
+          title='Developer Edition',
+          heading_level=2,
+          image_url='img/firefox/browsers/dev.png',
+          include_highres_image=True,
+          class='mzp-l-card-feature-left-half',
+          link_url=url('firefox.developer.index'),
+          link_cta=learn_more,
+          ga_title='Developer'
+        ) %}
+        <p class="c-dev-title mzp-has-zap-16">{{ _('Build sites and refine your code with Firefox <strong>DevTools</strong>') }}</p>
+        <p><a class="mzp-c-cta-link" href="{{ url('firefox.developer.index') }}">{{ _('Learn more about DevTools') }}</a></p>
+        {% endcall %}
+      </div>
+    </div>
+  </div>
+
+  <div class="mzp-l-content is-js-dependent">
+    <div class="mzp-c-emphasis-box mzp-t-dark t-account">
+      <div class="fxa-form-cta">
+        {{ fxa_email_form(
+          entrypoint=_entrypoint,
+          form_title=_('Join Firefox and get the most out of every product — across every device.'),
+          button_class='mzp-c-button mzp-t-product mzp-t-small'
+          )
+        }}
+        <p class="fxa-signin account-signin">
+          {% set fxa_link = fxa_link_fragment(entrypoint=_entrypoint, action='signin', utm_campaign='browsers_footer') %}
+          {% set fxa_attr = 'href="{{ fxa_link }}" class="js-fxa-cta-link"'|safe %}
+          {% set accounts_link = url('firefox.accounts') %}
+          {% set accounts_attr = 'href="{{ accounts_link }}"'|safe %}
```

Comment:
I'm not sure I understand why these are broken up into variables like this? It doesn't seem like these are used in other places in the template. Feels a little hard to read. 🤔

Reply:
Because I don't know how to include function calls inside of string variable assignments 😅 . I'm open to recommendations :)